libs/schemas/src/entities/group.schemas.ts

```diff
@@ -43,21 +43,11 @@ const ThresholdData = z.object({
   ]),
 });
 
-const AllowlistData = z.object({
```

Comment:
So, we don't support allow lists any more?

Reply:
After reading the ticket, it's not clear to me we want to remove the allow list schema.